src/System.Runtime/tests/Performance/Perf.String.netcoreapp.cs

```diff
@@ -10,15 +10,18 @@ public partial class Perf_String
 
         [Benchmark]
         [MemberData(nameof(ContainsStringComparisonArgs))]
-        public void Contains(StringComparison comparisonType, int size)
+        public void Contains(StringComparison comparisonType, int[] sizes)
```

Comment:
You are right, the array always contains a single element. On the other hand, this implementation will be correct even if the array contains more than one element.